code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1170,6 +1170,8 @@ GLOBAL_LIST_EMPTY(PDAs)
 	else
 		visible_message(span_danger("[src] explodes!"), span_warning("You hear a loud *pop*!"))
 
+	target.client.give_award(/datum/award/achievement/misc/clickbait, target)
```

Comment:
since clients can apparently go into the aether in the middle of executing code you need to use 
```suggestion
	target.client?.give_award(/datum/award/achievement/misc/clickbait, target)
```